FF13/DataStoreWDB.cs

```diff
@@ -24,6 +24,8 @@ public class DataStoreWDB<T> : DataStoreWDBEntry where T : DataStoreWDBEntry, ne
     public List<string> Keys => Entries.Keys.ToList();
     public List<T> Values => Entries.Values.ToList();
 
+    public List<KeyValuePair<string, T>> Entries => Data.Keys.Select(k => KeyValuePair.Create(k, Data[k])).ToList();
```

Comment:
I might need to change this now that you adjusted how some of the entries stuff works behind the scenes...

Reply:
yeah this would need to go through a refactor...hopefully for the better?